Documentation~/Selection_SelectByVertexColor.md

```diff
@@ -1,19 +1,28 @@
-# ![Select by Vertex Color icon](images/icons/Selection_SelectByVertexColor.png) Select by Colors
+# Select Vertex Color
 
-The __Select by Colors__ selects all faces on this object which have the same vertex color as the selected face. You can also extend the selection to other GameObjects if you disable the **Current Selection** option.
+To select all elements (faces, edges, or vertices) from faces that use the same vertex color, use the **Select Vertex Color** action. This action is available in the [vertex, edge, and face modes](modes.md).
 
-Even if the vertex color isn't currently visible (for example, if it has a Material that doesn't show colors, like the checkerboard Material), the colored faces are still selected.
+> **Note:** Selection works even on vertex colors that aren't currently visible, for example if their material shader doesn't display colors.
 
-![Select all orange faces on the Mesh](images/Example_SelectByVertexColor.png)
+![On the left, a single orange face is selected. On the right, all orange faces are selected.](images/Example_SelectByVertexColor.png)
 
-This action is useful for grouping out sections of your Mesh with different vertex colors. It is available in the [vertex, edge, and face modes](modes.md).
+To select faces by material:
 
-From the main menu, select **Tools** > **ProBuilder** > **Selection** > **Select Vertex Color**.
+1. In the **Tools** overlay, select the **ProBuilder** context.
+1. In the **Tool Settings** overlay, select an edit mode. All three modes support this action.
+1. Select an element. To select more than one color, hold **Shift** and select more elements. <!--this isn't as simple as I make it sound - when you select an edge or a vertex, it's not obvious which face it will reference for the color - it's honestly easier to use only Face mode-->
```

Comment:
We need to remove the code comments. They display in the published source of the output.

Reply:
I don't remove comments until right before we merge, unless they're dealt with.